### DISCUSSION TITLE
Player entitty velocity

### DISCUSSION THREAD

**Discussion by Cosmooor** [saul/demofile-net#62] [Q&A] [answered]
Hi,
Any chance to get velocity for player for each tick?

**Comment by in0finite** [comment, marked as answer]
Velocity of player is not stored in demos.

However there are some workarounds:

- you can calculate velocity yourself, based on current tick's position and previous tick's position (it's not ideal, and it's not equal to velocity as seen by server)

- I think I saw some AnimGraph parameters in demos that could be related to velocity
